source/diffHandler.py

```diff
@@ -62,6 +62,21 @@ def _initialize(self):
 	def _getText(self, ti: TextInfo) -> str:
 		return ti.text
 
+	@classmethod
+	def _readData(cls, size: int) -> bytes:
+		"""Reads from stdout, raises exception on EOF."""
+		buffer = b""
+		while (requiredLength := size - len(buffer)) > 0:

```

Comment:
Can you think of a better name than `requiredLength` for this? Maybe remaining instead of required?